### PR TITLE
Replace inflow and outflow fields in Transaction with amountMinor

### PR DESCRIPTION
**Pull request by jasonpetsod**
Add inflow() and outflow() methods that utilize amountMinor. Change components
to use these new methods.

Fix tests to not pass both inflow and outflow fields to Transaction().

Issue: #15

### CODE REVIEW

File: src/MutableTransactionRow.jsx [hunk 1]
```diff
@@ -35,8 +35,8 @@ export default class MutableTransactionRow extends React.Component {
         payee: this.props.initialTransaction.payee,
         category: this.props.initialTransaction.category,
         memo: this.props.initialTransaction.memo,
-        outflow: this.props.initialTransaction.outflow,
-        inflow: this.props.initialTransaction.inflow,
+        outflow: this.props.initialTransaction.outflow(),
```
Comment (witriew) [inline src/MutableTransactionRow.jsx:73]: Is it really being replaced if it's still being used?

**Review by witriew — COMMENTED** [review]
Maybe this should be pushed down to the translation layer that communicates with Firebase.

File: src/MutableTransactionRow.jsx (hunk 1 above)
Comment (jasonpetsod) [reply src/MutableTransactionRow.jsx:73]: Clarified in the new commit (5119bbd1b62f4beaf3df5045c9174eb1a8255116).

File: src/MutableTransactionRow.jsx [hunk 2]
```diff
@@ -22,6 +23,40 @@ export default class MutableTransactionRow extends React.Component {
     };
   }
 
+  static parseAmountMinor(inflowStr, outflowStr) {
+    const inflow = parseFloat(inflowStr);
+    if (inflowStr !== '' && isNaN(inflow)) {
```
Comment (witriew) [inline src/MutableTransactionRow.jsx:28]: isNaN(inflowStr)?

https://jsfiddle.net/d3sfjyt0/

File: src/Transaction.jsx [hunk 3]
```diff
@@ -2,20 +2,35 @@ import moment from 'moment';
...
+    this.amountMinor = amountMinor;
   }
 
   prettyDate() {
     // TODO: Do we need to store dates according to the user's desired
     // timezone?
     return moment(this.dateMs).utc().format('YYYY-MM-DD');
   }
+
+  inflow() {
```
Comment (witriew) [inline src/Transaction.jsx:21]: Aren't inflows and outflows supposed to be Decimal types?

File: src/MutableTransactionRow.jsx (hunk 2 above)
Comment (jasonpetsod) [reply src/MutableTransactionRow.jsx:28]: Yeah, addressed in PR #31.

File: src/Transaction.jsx (hunk 3 above)
Comment (jasonpetsod) [reply src/Transaction.jsx:21]: Yup, addressed in PR #31.

**Review by witriew — APPROVED** [review]

**Pull request merged**